$\require{cancel}  \require{bbox} \require{color} \require{enclose} \require{AMSmath} \require{AMSsymbols} \require{sinuitx} \require{mhchem}$
<div style="display:inline-block; position:absolute; top:0; right:0; color: #230e3e; font:italic 1em serif;">$\LaTeX$<span style="display:inline-block; vertical-align:bottom"><a href="https://www.mathjax.org">
    <img style="height:1.25em" title="Powered by MathJax"
    src="https://www.mathjax.org/badge/mj_logo.png"
    border="0" alt="Powered by MathJax" />
</a></span> </div>

<header style="background-color:#e7e2ee; color:#461d7c; line-height:2em; padding-left:1em; padding-right:1em; bottom:0; float:bottom; text-align:right; float:right; font-family:serif; font-size:small;">Problem 4-6 ⋮ Çengel, Y. A., and Boles, M. A. (2001). <i>Thermodynamics: An Engineering Approach.</i> Boston: McGraw-Hill.</header>  



## Problem 4-6

### Given   
A piston—cylinder device with a set of stops initially contains 0.6 kg of steam at 1.0 MPa and 400°C. The location of the stops corresponds to 40 percent of the initial volume. Now the steam is cooled.  


### Find 
Determine  
$\quad$ (a) $W$ for the final state of 1.0 MPa and 250 °C   
$\quad$ (b) $W, T_{final}$ for the final state of 500 kPa.  

### Get Organized

![figure, cb8, 4-6](./figures/ch04_cen98179_p04006_lg.jpg)

These things you should notice right away:
1. Closed system ⋮ constant mass.  
1. Isobaric process until the piston reaches the stops, isometric process after.   
1. We can resolve State 1 with $T_1, P_1$ (both given, and State 1 is _superheated vapor_).   
1. (a) Final state: 1.0 MPa and 250 °C: We can resolve State 2 with $P_2, T_2$ (given).
    ![pv diagram](./figures/cb8,4-006a.png)
1. (b) Final state: 500 kPa: 
    1. The piston will hit the stops
    2. We can resolve State 2 with $P_2$ (given) and $v_2$ (calculated).  
 
 ![pv diagram](./figures/cb8,4-006b.png)


Always, always, **always**: Find the state you can resolve. Relate that state to the states you need via the processes that connect them. Ride that Dog for the rest of your thermodynamic life. It will bring you Home every time.

#### (a) Find $W$ for the final state of 1.0 MPa and 250 °C 
All of our work is isobaric boundary work. 

\begin{gather}
   W_{b,out} = \int^2_1 P\,{\rm d}V = mP \left( v_2 - v_1 \right) \\
 \begin{aligned}
   v_1 &= v\big(\text{1 MPa, 400 °C}\big) \\
   v_2 &= v\big(\text{1 MPa, 250 °C}\big) \\   
   m &= 0.6\,\ce{kg} \\
   P &= \ce{1MPa} \\{}
 \end{aligned}
\end{gather}  


<div class="btn-info">&nbsp;<div class="badge"></div><i class="fa fa-exclamation-circle"></i><strong> Info: Symbolic Solution</strong></div>
<div class="panel-info"><div class="panel-heading">   

Were you solving this symbolically, you'd be done at this point. 

</div></div>


In [1]:
using Unitful

In [2]:
m  = 0.6u"kg"           # given
P  = 1u"MPa"            # given
v1 = 0.3066u"m^3/kg"    # table
v2 = 0.2327u"m^3/kg"    # table

W_out = m * P * (v2 - v1) |> u"kJ"

0.4*v1*m

0.073584 m^3

Incidentally, we can calculate the minimum volume:   

\begin{gather*}
  v_{min} = 0.4v_1 = 0.12264\,\ce{\frac{m^3}{kg}} \qquad \qquad
  V_{min} = mv_{min} = 0.07358\,\ce{m^3} \\{}
\end{gather*}


<div class="btn-success">&nbsp;<div class="badge"></div><i class="fa fa-bomb"></i><strong> You're the bomb</strong></div>
<div class="panel-success"><div class="panel-heading">   

$$(a)\ W_{in} = 44.34\,\ce{kJ}$$

</div>


#### (b) $W, T_{final}$ for the final state of 500 kPa.    
All of our work is _still_ isobaric boundary work, but now we know that the piston hits the stops. Remember there is no boundary work with no volume change. 

\begin{gather*}
   W_{b,out} = \int^{min}_1 P\,{\rm d}V = mP \big( v_{min} - v_1 \big) = mP \big( 0.4 v_{1} - v_1 \big) \\ 
   W_{b,out} = - 0.6mPv_1  \\[12pt]
 \begin{aligned}
   v_1 &= v\big(\text{1 MPa, 400 °C}\big) \\
   v_2 = v_{min} &= 0.4 v_1 \\
   m &= 0.6\,\ce{kg} 
 \end{aligned} \\[6pt]
   T_2 = T\big(\ce{500kPa},v=0.4v_1\big) \\{}
\end{gather*}  


<div class="btn-info">&nbsp;<div class="badge"></div><i class="fa fa-exclamation-circle"></i><strong> Info: Symbolic Solution</strong></div>
<div class="panel-info"><div class="panel-heading">   

Were you solving this symbolically, you'd be done at this point. 

</div></div>


In [3]:
m  = 0.6u"kg"           # given
P  = 1u"MPa"            # given
v1 = 0.3066u"m^3/kg"    # table
v2 = 0.4*v1       

W_out = m * P * (v2 - v1) |> u"kJ"


-110.37599999999999 kJ

<div class="btn-success">&nbsp;<div class="badge"></div><i class="fa fa-spin fa-fw fa-cog"></i><strong> Finis</strong></div>
<div class="panel-success"><div class="panel-heading"><div class="badge"></div> 

\begin{align*}
   (a) \ &\ W_{in} = 44.34\,\ce{kJ} \\[12pt]
   \begin{aligned}(b)\\{} \end{aligned} &\ \begin{aligned}
            W_{in} &= 110.376 \,\ce{kJ}\\
               T_2 &= 151.8\,\ce{°C}
          \end{aligned}
\end{align*}  


</div></div>

<footer style="background-color: #461d7c; color:rgba(253,208,35,0.35); line-height:2em; padding-left:1em; padding-right:0.5em; bottom:0; float:bottom; text-align:right;"><i>ψᵟ</i></footer>